apps/site/assets/ts/schedule/components/schedule-finder/ScheduleModalContent.tsx

```diff
@@ -125,6 +125,7 @@ const ScheduleModalContent = ({
           selectedDirection={selectedDirection}
           selectedOrigin={selectedOrigin}
           stopsByDirection={stops}
+          typeModal
```

Comment:
What does this mean? :eyes:

Reply:
Boolean to render type "modal" of ScheduleFinderForm.  All modal versions of ScheduleFinderForm should look the same (even for ferry) but the ferry version of ScheduleFinderForm that lives right on the page is the outlier to be styled differently.  I could make this value into a "className" but figured the boolean was... simpler?

Reply:
Is this different than what the `schedule-finder-vertical` class is already doing?

Reply:
Maybe a different name would make me less confused... like `insideModal` or something?

Reply:
Oh hey!  True -- I can do this all with CSS without too much headache.  One less inflexible prop on a component 👍